### PR TITLE
msx1_cart.xml: Added 37 items, 36 working.

### PR DESCRIPTION
**Pull request by wilbertpol**
New working software list items
-------------------------------
Game World - 64 Games (Korea, alt) [file-hunter]
Gangcheol Robocop (Korea) [file-hunter]
Gangcheol Robocop (Korea, alt) [file-hunter]
Gangcheol Robocop (Korea, alt 2) [file-hunter]
El Gnomo Fedor en la Caverna Perdida (Spain) [file-hunter]
Gokiburi Daisakusen (Japan, alt) [file-hunter]
Dunoegyosil Omok (Korea) [file-hunter]
Graphic Artist (Japan, Europe) [file-hunter]
Graphic Artist (Japan, Europe, alt) [file-hunter]
Yamaha Graphic Card [file-hunter]
Graphic Master Lab (Japan, Europe) [file-hunter]
Graphic Master Lab (Japan, Europe, alt) [file-hunter]
Green Beret (Europe, alt 2) [file-hunter]
Gyro Adventure (Japan) [file-hunter]
Nemesis 2 (Europe) ~ Gradius 2 (Japan) (demo, alt) [file-hunter]
G-Monkey [2KBOS]
Genesis Dawn of a New Day [Retroworks]
Genesis Dawn of a New Day (alt) [file-hunter]
Gniffel [MSXDev]
Goal [MSXDev]
Goal (alt) [file-hunter]
Gommy - Medieval Defender - Cartridge Edition [file-hunter]
Gommy - Medieval Defender (English) [MSXDev]
Gommy - Medieval Defender (Spanish) [MSXDev]
Gorgeous Gemma in Escape from the Space Disposal Planet [MSXDev]
Gorgeous Gemma in Escape from the Space Disposal Planet (older) [file-hunter]
GRAVITICA [MSXDev]
Green Cat Chronicles [N.I]
Grid Wars [MSXDev]
Griel's Quest for the Sangraal (extended edition) [MSXDev]
Griel's Quest for the Sangraal (extended edition, older) [file-hunter]
Griel's Quest for the Sangraal [MSXDev]
Griel's Quest for the Sangraal (older) [file-hunter] Guntus [Inufuto]
Guru Logic [file-hunter]
Guru Logic (alt) [file-hunter]

New NOT_WORKING software list additions
------------------------------------------
Gyruss [file-hunter]

### CODE REVIEW

File: hash/msx1_cart.xml [hunk 1]
```diff
@@ -5743,6 +5753,49 @@ kept for now until finding out what those bytes affect...
...
+		<publisher>Screen Software</publisher>
+		<info name="serial" value="SI-3131Z" />
+		<info name="alt_title" value="강철 로보캅" />
+		<part name="cart" interface="msx_cart">
+			<feature name="slot" value="konami" />
+			<dataarea name="rom" size="0x10000">
+				<rom name="gangjeol robocop (korea) (alt).rom" size="0x10000" crc="248dd0da" sha1="77cacb5e9a6cef48a36e8acabc938d5f1f1beadc" />
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/msx1_cart.xml:5783]: Something’s wrong here.  The Korean title says “강철” which should be transliterated as “Gangcheol” (I assume the Korean is correct, as “강철” means “steel” while “강절” doesn’t seem to be a real word).

File: hash/msx1_cart.xml [hunk 2]
```diff
@@ -5901,6 +5981,19 @@ kept for now until finding out what those bytes affect...
...
+		<year>1984</year>
+		<publisher>Zemina</publisher>
+		<info name="alt_title" value="두뇌교실 오목" />
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x4000">
+				<rom name="dunoegyosil omog (korea).rom" size="0x4000" crc="f4cb0a83" sha1="bd5617cfd9ee1d17f4cde6f282da897003226089" />
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/msx1_cart.xml:5995]: Correct RR for this title would be “Dunoegyosil Omok” (“ㄱ” is transliterated as “k” in final position where it’s pronounced as an unreleased stop).

File: hash/msx1_cart.xml (hunk 1 above)
Comment (wilbertpol) [reply hash/msx1_cart.xml]: Thanks, fixed.

File: hash/msx1_cart.xml (hunk 2 above)
Comment (wilbertpol) [reply hash/msx1_cart.xml]: Thanks, fixed.

**Pull request merged**